conf/defaults.yaml

```diff
@@ -193,11 +193,7 @@ backpressure.disruptor.low.watermark: 0.4
 backpressure.znode.timeout.secs: 30
 backpressure.znode.update.freq.secs: 15
 
-zmq.threads: 1
-zmq.linger.millis: 5000
```

Comment:
So why just remove zero-mq config? Is there any other deprecated conf need to be removed?